lib/ui/main.js

```diff
@@ -3,7 +3,7 @@ import BaconModel from 'bacon.model';
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
-import RGL, { WidthProvider } from "react-grid-layout";
+import RGL from "react-grid-layout";
```

Comment:
Why do we need our own copy?

Reply:
WidthProvider is provided in react-grid-layout as a template for a minimal implementation.
https://github.com/STRML/react-grid-layout#providing-grid-width.
I need to shrink the grid when in edit mode to be able to scroll more easily.
![image](https://user-images.githubusercontent.com/31212553/75633614-9ce47b00-5c06-11ea-8591-a90bc9b1c2e5.png)
![image](https://user-images.githubusercontent.com/31212553/75633616-a241c580-5c06-11ea-863b-31925be48d9b.png)